btree/btree.c

```diff
@@ -4365,6 +4278,8 @@ int64_t btree_close_tree_tick(struct m0_sm_op *smop)
 	case P_ACT:
 		tree->t_starttime = 0;
 		tree_put(tree);
+		M0_SET0(bop->bo_arbor->t_desc);
```

Comment:
Not needed. If you want to set tree descriptor to NULL after freeing, use m0_free0 in tree_put().

Reply:
Ok.
I will check this and update.